starters/nextjs-starter-approuter-ts/app/articles/[...uri]/article-view.tsx

```diff
@@ -55,7 +57,14 @@ export async function getServersideArticle({
 
   // Fetch the article and site in parallel
   const [article, site] = await Promise.all([
-    PCCConvenienceFunctions.getArticleBySlugOrId(slugOrId),
+    PCCConvenienceFunctions.getArticleBySlugOrId(
+      slugOrId,
+      {
```

Comment:
This doesn't throw a type error right now because the SDK version in the repo is the beta version. The stable release doesn't expect an object in the second param and so cloners of the starterkit depending on the stable version will get errors. Ref: [this commit](https://github.com/pantheon-systems/pantheon-content-cloud-sdk/pull/378/commits/226580a8af544e62fb80ded89915af05e7bda863) making this change.

I recommend just removing this diff until the next stable release goes out and the starterkit versions are bumped

Reply:
Resolved